<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gb_settings">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>471</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Настройки</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>387</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="fl_unitSettings">
      <item row="0" column="0">
       <widget class="QLabel" name="lb_baudRate">
        <property name="text">
         <string>Скорость</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cb_baudRate">
        <item>
         <property name="text">
          <string>115 200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9 600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1 200</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lb_dataBits">
        <property name="text">
         <string>Биты данных</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cb_dataBits">
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lb_parity">
        <property name="text">
         <string>Контроль четности</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cb_stopBits">
        <item>
         <property name="text">
          <string>no</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>even</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>odd</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>space</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>mark</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lb_stopBits">
        <property name="text">
         <string>Стоповые биты</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="cb_parity">
        <item>
         <property name="text">
          <string>1 bit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1.5 bit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2 bit</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lb_writeDelay">
        <property name="text">
         <string>Задержка между пакетами</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="ln_writeDelay"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lb_waitPacketTime">
        <property name="text">
         <string>Время ожидания данных</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="ln_waitPacketTime"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pb_cancel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>250</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_accept">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>250</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Применить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_edit">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>250</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Изменить</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lb_status">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>490</y>
      <width>44</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_finishConfigure">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Завершить настройку МПД</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gb_start">
    <property name="geometry">
     <rect>
      <x>179</x>
      <y>89</y>
      <width>221</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Выбор порта</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>151</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="vl_start">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="cb_avaliablePorts"/>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pb_startConfigure">
        <property name="text">
         <string>Подсоединить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb_model">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>471</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pb_radio">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Radio</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_mpd">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>151</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string>МПД</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_rs232">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Rs-232</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_rs485">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>80</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Rs-485</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_disconnect">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>120</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Отсоединить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_disconnect_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Задействовать</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>618</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
